web/packages/build/.eslintrc.js

```diff
@@ -65,6 +65,14 @@ module.exports = {
         'jest/no-large-snapshots': ['warn', { maxSize: 200 }],
       },
     },
+    // Allow require imports in .js files, as migrating our project to ESM modules requires a lot of
+    // changes.
+    {
+      files: ['**/*.js'],
+      rules: {
+        '@typescript-eslint/no-require-imports': 'warn',
```

Comment:
I tried replacing requires with imports everywhere (it's on `r7s/esm`), but I couldn't get Babel to work with Jest.

Reply:
I currently have a branch trying to upgrade eslint to v9 and all the dependencies have required a massive change. We can table this one until I'm able to carve out enough time this quarter to finally tackle it.

Reply:
Wouldn't it be better if this particular update was already taken care of? Then the final PR will be smaller.

Reply:
And we wouldn't risk introducing new lint issues.